docs/0.1/creating_products.md

```diff
@@ -0,0 +1,313 @@
+`grid` commands in this procedure.
+
+{:start="2"}
+
+2. Set `GRID_DAEMON_KEY` to the base name of your public/private key files
+   (the example environment uses `alpha-agent`).  This environment variable
+   replaces the `-k` option on the `grid` command line.
+
+   ```
+   root@gridd-alpha:/# export GRID_DAEMON_KEY="alpha-agent"
+   ```
+
+   **Note**: Although this variable has "daemon" in the name, it should
+   reference the ***user key files*** in `$HOME/.grid/keys`, not the Grid
+   daemon's key files in `/etc/grid/keys`.
+
+1. Set `GRID_DAEMON_ENDPOINT` to the Grid daemon's endpoint (URL and port),
+   such as `http://localhost:8080`. This environment variable replaces the
+   `-url` option on the `grid` command line.
+
+   ```
+   root@gridd-alpha:/# export GRID_DAEMON_ENDPOINT="http://localhost:8080"
+   ```
+
+1. Set `GRID_SERVICE_ID` to the fully qualified service ID (such as
+   `01234-ABCDE::xyZZ`) for the scabbard service on the circuit. This
+   environment variable replaces the `--service_id` option on the `grid`
+   command line.
+
+   ```
+   root@gridd-alpha:/# export GRID_SERVICE_ID="01234-ABCDE::xyZZ"
+   ```
+
+   Tip: See [Prerequisites](#prerequisites) for the `splinter` commands that
+   display the circuit ID and four-character service ID string.
+
+### Define and Create a Product
+
+Creating a product has two steps. First, create a product definition file in
+YAML format. Then run `grid product create` to create the product based on this
+definition.
+
+**Note**: Each product requires an organization ID (the owner) and at least one
+agent with permission to create, update, and delete the product. See
+[Prerequisites](#prerequisites) for the commands to display the existing
+organizations and agents.
+
+{:start="5"}
+
+5. Create a product definition file, in YAML format, that specifies the
+   following information:
+
+   * Product identification type (such as GS1)
+   * Product ID (such as a GTIN)
+   * Owner (organization ID)
+   * Set of product properties (each with a name, data type, and value
+     that conforms to the data type)
+     <br><br>
+
+   For example, you can use `cat` to create the file `product.yaml`, using the
+   following contents.
+
+   ```
+   root@gridd-alpha:/# cat > product.yaml
+   - product_type: "GS1"
+     product_id: "723382885088"
+     owner: "314156"
+     properties:
+       - name: "species"
+         data_type: "STRING"
+         string_value: "tuna"
+       - name: "length"
+         data_type: "NUMBER"
+         number_value: 22
+       - name: "maximum_temperature"
+         data_type: "NUMBER"
+         number_value: 5
+       - name: "minimum_temperature"
+         data_type: "NUMBER"
+         number_value: 0
+   ```
+
+   Tip: Enter CTRL-C to exit the file and save the contents.
+
+1. Add the new product by using the `grid product create` command to specify the
+   definition in the `product.yaml` file.
+
+   ```
+   root@gridd-alpha:/# grid product create product.yaml
```

Comment:
This now fails because of missing schema when following along with the steps in the grid on splinter steps.

Reply:
adding the schema be added to prereqs